components/schemes/Color/src/MDCSemanticColorScheme.h

```diff
@@ -0,0 +1,144 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import "MDCColorScheme.h"
+
+/**
+ A simple color scheme that provides semantic context for the colors it uses. There are no optional
+ properties and all colors must be provided, supporting more reliable color theming.
+ */
+@protocol MDCColorScheming
+
+/**
+ Displayed most frequently across your app.
+ */
+@property(nonnull, readonly, nonatomic) UIColor *primaryColor;
+
+/**
+ A tonal variation of primary color.
+ */
+@property(nonnull, readonly, nonatomic) UIColor *primaryColorVariant;
+
+/**
+ Accents select parts of your UI.
+ */
+@property(nonnull, readonly, nonatomic) UIColor *secondaryColor;
```

Comment:
secondaryColorVariant is listed in go/m2-color-overview.
Not here?

Reply:
It wasn't in pivotal, I leave the doc to the authors.

Reply:
We'll add these parameters in a follow-up pass once the parameter mappings are more clearly defined.